model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -158,6 +159,39 @@ class OrtConfigurationTest : WordSpec({
 
             config.scanner.shouldBeNull()
         }
+
+        "support references to environment variables" {
+            val configFile = createTestConfig(
+                """
+                ort {
+                  scanner {
+                    storages {
+                      postgresStorage {
+                        url = "postgresql://your-postgresql-server:5444/your-database"
+                        schema = schema
+                        username = ${'$'}{POSTGRES_USER}
+                        password = ${'$'}{POSTGRES_PWD}
```

Comment:
Nit: "PWD" sounds a bit like the "Print Working Directory" environment variable; better use `POSTGRES_PASSWORD`, and while at it, also `POSTGRES_USERNAME` (like the key name).